### PR TITLE
Add SetRadiantCrystals() and SetEbonCrystals() to Perl/Lua.

### PR DESCRIPTION
**Pull request by KinglyKrab**
- Add $client->SetRadiantCrystals(value) to Perl.
- Add $client->SetEbonCrystals(value) to Perl.
- Add client:SetRadiantCrystals(value) to Lua.
- Add client:SetEbonCrystals(value) to Lua.

### CODE REVIEW

File: zone/perl_client.cpp [hunk 1]
```diff
@@ -4707,6 +4707,52 @@ XS(XS_Client_AddCrystals) {
...
+		if (sv_derived_from(ST(0), "Client")) {
+			IV tmp = SvIV((SV *) SvRV(ST(0)));
+			THIS = INT2PTR(Client *, tmp);
+		} else
+			Perl_croak(aTHX_ "THIS is not of type Client");
+		if (THIS == nullptr)
+			Perl_croak(aTHX_ "THIS is nullptr, avoiding crash.");
+
+		THIS->SetEbonCrystals(value);
+		THIS->SendCrystalCounts();
```
Comment (Akkadius) [inline zone/perl_client.cpp:4728]: This is redundant since `SetEbonCrystals` sends counts

File: zone/perl_client.cpp [hunk 2]
```diff
@@ -4707,6 +4707,52 @@ XS(XS_Client_AddCrystals) {
...
+		if (sv_derived_from(ST(0), "Client")) {
+			IV tmp = SvIV((SV *) SvRV(ST(0)));
+			THIS = INT2PTR(Client *, tmp);
+		} else
+			Perl_croak(aTHX_ "THIS is not of type Client");
+		if (THIS == nullptr)
+			Perl_croak(aTHX_ "THIS is nullptr, avoiding crash.");
+
+		THIS->SetRadiantCrystals(value);
+		THIS->SendCrystalCounts();
```
Comment (Akkadius) [inline zone/perl_client.cpp:4751]: This is redundant since `SetRadiantCrystals` sends counts

**Review by Akkadius — CHANGES_REQUESTED** [review]
Looks good, just a few comments

**Comment by KinglyKrab** [comment]
Removed my extraneous debug code from when I first wrote it.

**Review by Akkadius — APPROVED** [review]
Nice work!

**Pull request merged**